File: app/src/main/java/com/doubean/ford/data/SizedImage.java
package com.doubean.ford.data;

import com.google.gson.annotations.SerializedName;

public class SizedImage {
    @SerializedName("is_animated")
    public boolean isAnimated;
    public ImageItem large;
    public ImageItem normal;
    public ImageItem raw;
    public ImageItem small;
    public ImageItem video;

    public static class ImageItem {
        public int height;
        public long size;
        public String url;
        public int width;
    }
}

File: app/src/main/java/com/doubean/ford/data/vo/SizedImage.kt
package com.doubean.ford.data.vo

import com.google.gson.annotations.SerializedName

data class SizedImage (
    @SerializedName("is_animated")
    val isAnimated: Boolean = false,
    val large: ImageItem? = null,
    val normal: ImageItem? = null,
    val raw: ImageItem? = null,
    val small: ImageItem? = null,
    val video: ImageItem? = null
)
    data class ImageItem (
        val height: Int = 0,
        val size: Long = 0,
        val url: String? = null,
        val width: Int = 0
    )
